src/components/ProfileButton/ProfileButton.component.tsx

```diff
@@ -1,35 +1,50 @@
 import React, { useState, useRef } from 'react';
-import { BsPersonFill } from 'react-icons/bs';
+import { useHistory } from 'react-router-dom';
 import 'react-toastify/dist/ReactToastify.min.css';
 
+import { useAuth } from '../../providers/Auth';
 import useOnClickOutside from '../../hooks/useOnClickOutside';
 import { ProfileIconWrapper, Dropdown } from './ProfileButton.styled';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.min.css';
 
 function ProfileButton() {
+  const { logout } = useAuth();
   const [openDropdown, setOpenDropdown] = useState(false);
```

Comment:
Let's separate useState variables from hooks. It's easier to maintain in my opinion. Feel free to ignore this!